tests/sqs_service.py

```diff
@@ -1,9 +1,10 @@
 import requests
 
-SQS_STAGE = "https://sqs.stage.osmosis.zone"
+SQS_STAGE = "https://sqs-eu.stage.osmosis.zone"
```

Comment:
Just curious - why specifically pointing to eu ?

Reply:
Was debugging stage. Going to revert now - thanks for pointing out